packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -626,6 +638,17 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
         }
     };
 
+    React.useEffect(() => {
+        let timeout: NodeJS.Timeout;
+        if (is_incorrect_mt5_password_format_error) {
+            timeout = setTimeout(() => {
```

Comment:
Can you please add a comment stating the timeout is required for rendering the modals. 
Will be helpful in understanding the usecase here

Reply:
Sure